### ISSUE TITLE
When I set alpah parameter to 1, some colors lose

### ISSUE THREAD

**Issue by gonghao** [seanooi/iOS-WebP#7]
I met an strange question which is that I compressed an UIImage to WebP, but some colors lose, and I also try to use other quality and WEBP_PRESET but fianlly I find the problem is casued by ALPHA parameter. I tried use other numbers less than 1 all are correct. And I try to edit some source code:

```
if (alpha <= 1) {
    image = [self webPImage:image withAlpha:alpha];
}
```

to enforce call `webPImage:withAlpha` method first and will fix this problem. I don't why, so could you help to fix this issue?

THX

**Comment by thegamingart** [comment]
Yeah, same thing is happening to me. Removing the if alpha <= 1 clause corrects the issue when the alpha is one. Otherwise, I consistently get a blue hue on all converted images.
